src/wristpy/core/cli.py

```diff
@@ -0,0 +1,92 @@
+"""CLI for wristpy."""
+
+import argparse
+import pathlib
+from typing import List, Optional
+
+from wristpy.core import config, orchestrator
+
+logger = config.get_logger()
+settings = config.Settings()
+
+
+def parse_arguments(args: Optional[List[str]] = None) -> argparse.Namespace:
+    """Argument parser for python cli.
+
+    Args:
+        args: Optional argument for when accessed via python script.
+
+    Returns:
+        Namespace object with all of the input arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Run the main wristpy pipeline",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument("input", type=pathlib.Path, help="Path to the input data.")
+
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=pathlib.Path,
+        help="Optional path where data will be saved.",
+    )
+
+    parser.add_argument(
```

Comment:
I'm gonna do an annoying teaching moment here: this argument contains a bug. If you resolve the smoke test to a unit test (c.f. comments on that file) you should catch it.

Reply:
Is it related to my comment above ? The line is now: 

```
parser.add_argument(
        "-c",
        "--calibrator",
        type=str,
        choices=["ggir", "gradient"],
        help="Pick which calibrator to use. Can be 'ggir' or 'gradient'.",
    )
```